omnisafe/algos/model_based/lagrange.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Lagrange"""
+
+import abc
+
+import torch
+import torch.optim as optim
+
+
+class Lagrange(abc.ABC):
```

Comment:
why need another Lagrange, what is the difference between it and `common/lagrange.py`